src/scripts/parseDateSearchParam.ts

```diff
@@ -0,0 +1,40 @@
+import * as fs from "fs";
```

Comment:
I noticed that this script doesn't have a "Wrote to..." logger. Do we want to add that?